docs/source/config/intro.rst

```diff
@@ -112,6 +112,18 @@ At present, this only affects the current session - changes you make to
 config are not saved anywhere. Also, some options are only read when
 IPython starts, so they can't be changed like this.
 
+.. _configure_embedded:
+
+Configuring embedded IPython
```

Comment:
Yeah, we added that because we found that quite a few people were embed()-ing when they didn't really need that, because we hadn't made start_ipython() obvious before. It's meant to be an 'instead of embedding, try this...' note, but it does make it a bit confusing what we actually mean by embedding.

"Starting IPython from Python" sounds good to me. :-)